packages/vite/src/node/plugins/css.ts

```diff
@@ -1780,15 +1914,19 @@ async function minifyCSS(
       ...config.css?.lightningcss,
       targets: convertTargets(config.build.cssTarget),
       cssModules: undefined,
-      filename: cssBundleName,
+      filename: filename || cssBundleName,
```

Comment:
```suggestion
      filename: filename,
```
`filename` always exists now.